filters/cropbyfocalpoint_test.go

```diff
@@ -0,0 +1,125 @@
+package filters
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"github.com/zalando-stups/skrop/filters/imagefiltertest"
+	"github.com/zalando/skipper/filters"
+)
+
+func TestNewCropByFocalPoint(t *testing.T) {
+	name := NewCropByFocalPoint().Name()
+	assert.Equal(t, "cropByFocalPoint", name)
+}
+
+func TestCropByFocalPoint_Name(t *testing.T) {
+	c := cropByFocalPoint{}
+	assert.Equal(t, "cropByFocalPoint", c.Name())
+}
+
+func TestCropByFocalPoint_CreateOptions(t *testing.T) {
+	c := cropByFocalPoint{targetX: 0.5, targetY: 0.5, aspectRatio: 1.5}
```

Comment:
Probably these repeating tests could be moved to a TableDriven tests?

Reply:
Given my very limited Go experience I wouldn't disagree if you suggest that.

I'll have a look on how to do this.

Reply:
Great. I am new too, but would love to be of help on this :)